app/src/main/java/com/github/polybooks/core/Books.kt

```diff
@@ -43,5 +43,6 @@ enum class BookFields(val fieldName: String) {
     LANGUAGE("language"),
     PUBLISHER("publisher"),
     PUBLISHDATE("publishDate"),
-    FORMAT("format")
+    FORMAT("format"),
+    PICTUREFILENAME("pictureFileName")
```

Comment:
Is there a reason for this? It doesn't seem to be used and doesn't reflect a field of book

Reply:
Will we have pictures for Books or only exclusively for Sales?
I thought that at some point we considered having pictures for books too, but maybe we didn't stick to that decision.